### ISSUE TITLE
.kml file support   + current   

### ISSUE THREAD

**Issue by ImreSamu** [julien-noblet/download-geofabrik#20]
//  not critical ; just for brainstorming  :) 
 
with a simple hack I can download a `.kml` file  (  instead of .poly ) 

```
/tools/download-geofabrik update
sed -i 's/ext: poly/ext: kml/g'   geofabrik.yml
sed -i 's/loc: .poly/loc: .kml/g' geofabrik.yml
/tools/download-geofabrik download -p hungary 
mv hungary.poly hungary.kml
head hungary.kml
<?xml version="1.0" encoding="UTF-8"?> <kml xmlns="http://earth.google.com/kml/2.0"> <Document> <Placemark> <MultiGeometry>
<Polygon> <outerBoundaryIs> <LinearRing>  <coordinates>
16.112620,46.869210,0
16.118770,46.872960,0
16.133400,46.878800,0
16.137210,46.885380,0
16.147690,46.893230,0
16.159120,46.900850,0
16.162460,46.905690,0
16.181830,46.916910,0
```

I don't know the .kml download is useful or not for the other users.

**Comment by julien-noblet** [comment]
Yeah! :+1: 
Could you try [v2.2.0-beta-fix#20](https://github.com/julien-noblet/download-geofabrik/releases/tag/v2.2.0-beta-fix%2320) ?

**Comment by ImreSamu** [comment]
perfect :)
Thank you!    

```bash
 ./download-geofabrik download -k hungary
2018/03/17 19:03:21 hungary.kml  downloaded.
root@84ab18b10284:/tools# head hungary.kml
<?xml version="1.0" encoding="UTF-8"?> <kml xmlns="http://earth.google.com/kml/2.0"> <Document> <Placemark> <MultiGeometry>
<Polygon> <outerBoundaryIs> <LinearRing>  <coordinates>
16.112620,46.869210,0
16.118770,46.872960,0
16.133400,46.878800,0
16.137210,46.885380,0
16.147690,46.893230,0
16.159120,46.900850,0
16.162460,46.905690,0
16.181830,46.916910,0
```
